packages/api/lambdas/sqs-message-consumer.js

```diff
@@ -64,7 +64,6 @@ async function processQueues(event, dispatchFn) {
     log.info(`processing queue ${queueUrl}`);
 
     return consumer.consume(dispatchFn.bind({
-      queueUrl,
       rulesForQueue,
     }));
   }));
```

Comment:
Is there a reason why writing it like this would be unpreferred?
```suggestion
    const messageConsumerFn = dispatchFn.bind({ rulesForQueue });
    return consumer.consume(messageConsumerFn);
```

Reply:
Not as far as I know. I can change it

Reply:
9e0d5de43fdbfde6900be393345030b2d8328d8a